src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/CliHelpers/DotnetCliRunner.cs

```diff
@@ -2,12 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 using System.Diagnostics;
 
-namespace Microsoft.DotNet.Scaffolding.Helpers.General;
+namespace Microsoft.DotNet.Scaffolding.Core;
 
 /// <summary>
 /// To run 'dotnet' or any processes and capture consequent stdout and stderr (using 'ExecuteAndCaptureOutput'
 /// </summary>
-internal class DotnetCliRunner
+public class DotnetCliRunner
```

Comment:
Do you intend to move this to the .Internal assembly in a second refactoring PR? I'm not sure about having it be part of the public API here so wanted to check.

Reply:
Actually it should be in `Internal` and Core might reference it as a `Compile Include` since Core won't be referencing Internal